embark-org.el

```diff
@@ -289,10 +288,8 @@ what part or in what format the link is copied."
 (embark-org-define-link-copier description description "'s description")
 (embark-org-define-link-copier target target "'s target")
 
-(declare-function embark-org-copy-link-inner-target "embark-org")
-(fset 'embark-org-copy-link-inner-target 'kill-new)
-(put 'embark-org-copy-link-inner-target 'function-documentation
-      "Copy 'inner part' of the Org link at point's target.
+(defalias 'embark-org-copy-link-inner-target #'kill-new
+  "Copy 'inner part' of the Org link at point's target.
```

Comment:
Ha! I didn't know `defalias`  took a docstring too! I would have used it had I known.